test/units/plugins/action/test_synchronize.py

```diff
@@ -99,11 +99,11 @@ class SynchronizeTester(object):
     execute_called = False
 
 
-    def _execute_module(self, module_name, task_vars=None):
+    def _execute_module(self, module_name, module_args=None, task_vars=None):
```

Comment:
Is this change needed?  It doesn't seem wrong, necessarily... I'm just wondering whether it is necessary and if so if there's something that should be refactored at a later date.